docs/cas-server-documentation/protocol/CAS-Protocol-Specification.md

```diff
@@ -290,17 +293,15 @@ handled by `/login`.
 ### **2.2.2. parameters for username/password authentication**
 
 In addition to the OPTIONAL parameters specified in Section [2.2.1](#head2.2.1),
-the following HTTP request parameters MUST be passed to `/login` while it is
+the following HTTP request parameters are to be passed to `/login` while it is
```

Comment:
Revert this please.

Reply:
OK, done. I was thinking to myself that I maybe overdid this. :)